packages/tpetra/core/test/KokkosIntegration/KokkosDeepcopy.cpp

```diff
@@ -240,17 +248,28 @@ class DeepCopyTester {
 
 private:
   bool isConsistent;
-  void log(const std::string& msg) {
+  void log(const std::string& msg, ApiTest *counter=0) {
```

Comment:
I'd strongly prefer to use nullptr as the default

Reply:
Sure, why not.